pkg/controller/controller.go

```diff
@@ -731,8 +730,6 @@ func instancesInIsoGroup(pl m3placement.Placement, isoGroup string) []m3placemen
 	return insts
 }
 
-// This func is currently read-only, but if we end up modifying statefulsets
```

Comment:
Removed this comment since we are calling `DeepCopy` before adding a StatefulSet to the slice that we return.